docs/ref/models/fields.txt

```diff
@@ -1116,7 +1116,7 @@ Django also defines a set of fields that represent relations.
 ``ForeignKey``
 --------------
 
-.. class:: ForeignKey(othermodel, [**options])
+.. class:: ForeignKey(othermodel, on_delete, [**options])
```

Comment:
Unless we keep it as a keyword arg to force it to be specified explicitly (and raise an exception manually if it's missing). I don't really have a strong opinion. I tend to think passing it explicitly will be preferred due to apps wanting to stay backwards compatible.